### PR TITLE
URI to URL

### LINKED ISSUE #11450: The mozbrowserlocationchanged event property should be url, not uri
The documentation at https://developer.mozilla.org/en-US/docs/Web/Events/mozbrowserlocationchange describes the details object has having property:

**url** The URL of the new location

but our implementation https://github.com/servo/servo/blob/master/components/script/dom/webidls/BrowserElement.webidl#L66 has a property **uri**.

### PR DESCRIPTION
**Pull request by Coder206**
---

- [x] `./mach build -d` does not report any errors
- [x] `./mach test-tidy` does not report any errors
- [x] These changes fix #11450 (github issue number if applicable).

- [x] There are tests for these changes (Windows 10 was never able to run them...)

---

This change is [Reviewable](https://reviewable.io/reviews/servo/servo/11789)

### CODE REVIEW

**Comment by Coder206** [comment]
@asajeffrey I think I got the hang of Git now.

**Comment by Coder206** [comment]
@emilio I have been having trouble with Git so this PR replaces #11782 and all the others like it.

**Comment by Coder206** [comment]
@Wafflespeanut Did I PR correctly this time?

**Comment by Wafflespeanut** [comment]
Yep! Good work! Feel free to ping us at IRC when you have trouble with something :smile:

**Comment by Coder206** [comment]
@Wafflespeanut Thanks! I actually just recreated my work space for Servo on my computer to un-merge from the master branch.

**Comment by jdm** [comment]
The tests `redirect.html` and `iframe_goback.html` in `tests/wpt/mozilla/tests/mozilla/mozbrowser` will need to be updated. The only other failure in the log is #11574.

File: tests/wpt/mozilla/tests/mozilla/mozbrowser/iframe_goback.html [hunk 1]
```diff
@@ -19,7 +19,7 @@
   iframe.src = url1;
 
   iframe.addEventListener("mozbrowserlocationchange", e => {
-    locations.push(e.detail.uri);
+    locations.push(e.detail.url);
     if (e.detail.uri == url2) {
```
Comment (jdm) [inline tests/wpt/mozilla/tests/mozilla/mozbrowser/iframe_goback.html]: This line didn't get updated.

**Comment by Coder206** [comment]
@jdm Thanks :)

**Comment by Coder206** [comment]
@jdm That is what I was trying to see! :)

File: tests/wpt/mozilla/tests/mozilla/mozbrowser/iframe_goback.html (hunk 1 above)
Comment (Coder206) [inline tests/wpt/mozilla/tests/mozilla/mozbrowser/iframe_goback.html]: On it!

**Comment by KiChjang** [comment]
That's #11703.

**Comment by nox** [comment]
Please squash the commits together.

**Comment by Coder206** [comment]
I cannot run the tests:

```
$ ./mach test-wpt --release --ssl-type=none tests/wpt/mozilla/tests/mozilla/mozbrowser
Running 14 tests in web-platform-tests

Unsupported test type wdspec for product servo
  [0/14] /_mozilla/mozilla/mozbrowser/crossorigin_xhr.html
      1 [main] python2.7 5428 fork: child -1 - forked process 4696 died unexpectSTDERR: ['/e/files/dev/mozilla/servo/target/release/servo', '--cpu', '--hard-fail', '-u', 'Servo/wptrunner', '-Z', 'replace-surrogates', '-z', u'http://web-platform.test:8000/_mozilla/mozilla/mozbrowser/crossorigin_xhr.html', '--user-stylesheet', '/e/files/dev/mozilla/servo/resources/ahem.css', '--pref', u'dom.mozbrowser.enabled=true']
  [0/14] /_mozilla/mozilla/mozbrowser/crossorigin_xhr.html
         /_mozilla/mozilla/mozbrowser/iframe_goback.html
  ▶ ERROR [expected OK] /_mozilla/mozilla/mozbrowser/crossorigin_xhr.html
  └   → Traceback (most recent call last):
  File "/e/files/dev/mozilla/servo/tests/wpt/harness/wptrunner/executors/base.py", line 149, in run_test
    result = self.do_test(test)
  File "/e/files/dev/mozilla/servo/tests/wpt/harness/wptrunner/executors/executorservo.py", line 100, in do_test
    self.proc.run()
  File "/e/files/dev/mozilla/servo/python/_virtualenv/lib/python2.7/site-packages/mozprocess/processhandler.py", line 700, in run
    self.proc = self.Process([self.cmd] + self.args, **args)
  File "/e/files/dev/mozilla/servo/python/_virtualenv/lib/python2.7/site-packages/mozprocess/processhandler.py", line 103, in __init__
    universal_newlines, startupinfo, creationflags)
  File "/usr/lib/python2.7/subprocess.py", line 710, in __init__
    errread, errwrite)
  File "/usr/lib/python2.7/subprocess.py", line 1231, in _execute_child
    self.pid = os.fork()
OSError: [Errno 11] Resource temporarily unavailable


STDERR: ['/e/files/dev/mozilla/servo/target/release/servo', '--cpu', '--hard-fail', '-u', 'Servo/wptrunner', '-Z', 'replace-surrogates', '-z', u'http://web-platform.test:8000/_mozilla/mozilla/mozbrowser/iframe_goback.html', '--user-stylesheet', '/e/files/dev/mozilla/servo/resources/ahem.css', '--pref', u'dom.mozbrowser.enabled=true']
  [1/14] /_mozilla/mozilla/mozbrowser/iframe_reload_twice.html
         /_mozilla/mozilla/mozbrowser/iframe_goback.html
  ▶ ERROR [expected OK] /_mozilla/mozilla/mozbrowser/iframe_goback.html
  └   → Traceback (most recent call last):
  File "/e/files/dev/mozilla/servo/tests/wpt/harness/wptrunner/executors/base.py", line 149, in run_test
    result = self.do_test(test)
  File "/e/files/dev/mozilla/servo/tests/wpt/harness/wptrunner/executors/executorservo.py", line 100, in do_test
    self.proc.run()
  File "/e/files/dev/mozilla/servo/python/_virtualenv/lib/python2.7/site-packages/mozprocess/processhandler.py", line 700, in run
    self.proc = self.Process([self.cmd] + self.args, **args)
  File "/e/files/dev/mozilla/servo/python/_virtualenv/lib/python2.7/site-packages/mozprocess/processhandler.py", line 103, in __init__
    universal_newlines, startupinfo, creationflags)
  File "/usr/lib/python2.7/subprocess.py", line 710, in __init__
    errread, errwrite)
  File "/usr/lib/python2.7/subprocess.py", line 1231, in _execute_child
    self.pid = os.fork()
OSError: [Errno 11] Resource temporarily unavailable


STDERR: ['/e/files/dev/mozilla/servo/target/release/servo', '--cpu', '--hard-fail', '-u', 'Servo/wptrunner', '-Z', 'replace-surrogates', '-z', u'http://web-platform.test:8000/_mozilla/mozilla/mozbrowser/iframe_reload_twice.html', '--user-stylesheet', '/e/files/dev/mozilla/servo/resources/ahem.css', '--pref', u'dom.mozbrowser.enabled=true']
  [2/14] /_mozilla/mozilla/mozbrowser/iframe_visibility.html
         /_mozilla/mozilla/mozbrowser/iframe_reload_twice.html
  ▶ ERROR [expected OK] /_mozilla/mozilla/mozbrowser/iframe_reload_twice.html
  └   → Traceback (most recent call last):
  File "/e/files/dev/mozilla/servo/tests/wpt/harness/wptrunner/executors/base.py", line 149, in run_test
    result = self.do_test(test)
  File "/e/files/dev/mozilla/servo/tests/wpt/harness/wptrunner/executors/executorservo.py", line 100, in do_test
    self.proc.run()
  File "/e/files/dev/mozilla/servo/python/_virtualenv/lib/python2.7/site-packages/mozprocess/processhandler.py", line 700, in run
    self.proc = self.Process([self.cmd] + self.args, **args)
  File "/e/files/dev/mozilla/servo/python/_virtualenv/lib/python2.7/site-packages/mozprocess/processhandler.py", line 103, in __init__
    universal_newlines, startupinfo, creationflags)
  File "/usr/lib/python2.7/subprocess.py", line 710, in __init__
    errread, errwrite)
  File "/usr/lib/python2.7/subprocess.py", line 1231, in _execute_child
    self.pid = os.fork()
OSError: [Errno 11] Resource temporarily unavailable


  [3/14] /_mozilla/mozilla/mozbrowser/iframe_visibility.html
      1 [main] python2.7 1244 fork: child -1 - forked process 3884 died unexpectSTDERR: ['/e/files/dev/mozilla/servo/target/release/servo', '--cpu', '--hard-fail', '-u', 'Servo/wptrunner', '-Z', 'replace-surrogates', '-z', u'http://web-platform.test:8000/_mozilla/mozilla/mozbrowser/iframe_visibility.html', '--user-stylesheet', '/e/files/dev/mozilla/servo/resources/ahem.css', '--pref', u'dom.mozbrowser.enabled=true']
STDERR: ['/e/files/dev/mozilla/servo/target/release/servo', '--cpu', '--hard-fail', '-u', 'Servo/wptrunner', '-Z', 'replace-surrogates', '-z', u'http://web-platform.test:8000/_mozilla/mozilla/mozbrowser/mozbrowser_click_fires_openwindow.html', '--user-stylesheet', '/e/files/dev/mozilla/servo/resources/ahem.css', '--pref', u'dom.mozbrowser.enabled=true']
  ▶ ERROR [expected OK] /_mozilla/mozilla/mozbrowser/iframe_visibility.html
  └   → Traceback (most recent call last):
  File "/e/files/dev/mozilla/servo/tests/wpt/harness/wptrunner/executors/base.py", line 149, in run_test
    result = self.do_test(test)
  File "/e/files/dev/mozilla/servo/tests/wpt/harness/wptrunner/executors/executorservo.py", line 100, in do_test
    self.proc.run()
  File "/e/files/dev/mozilla/servo/python/_virtualenv/lib/python2.7/site-packages/mozprocess/processhandler.py", line 700, in run
    self.proc = self.Process([self.cmd] + self.args, **args)
  File "/e/files/dev/mozilla/servo/python/_virtualenv/lib/python2.7/site-packages/mozprocess/processhandler.py", line 103, in __init__
    universal_newlines, startupinfo, creationflags)
  File "/usr/lib/python2.7/subprocess.py", line 710, in __init__
    errread, errwrite)
  File "/usr/lib/python2.7/subprocess.py", line 1231, in _execute_child
    self.pid = os.fork()
OSError: [Errno 11] Resource temporarily unavailable


  ▶ ERROR [expected OK] /_mozilla/mozilla/mozbrowser/mozbrowser_click_fires_openwindow.html
  └   → Traceback (most recent call last):
  File "/e/files/dev/mozilla/servo/tests/wpt/harness/wptrunner/executors/base.py", line 149, in run_test
    result = self.do_test(test)
  File "/e/files/dev/mozilla/servo/tests/wpt/harness/wptrunner/executors/executorservo.py", line 100, in do_test
    self.proc.run()
  File "/e/files/dev/mozilla/servo/python/_virtualenv/lib/python2.7/site-packages/mozprocess/processhandler.py", line 700, in run
    self.proc = self.Process([self.cmd] + self.args, **args)
  File "/e/files/dev/mozilla/servo/python/_virtualenv/lib/python2.7/site-packages/mozprocess/processhandler.py", line 103, in __init__
    universal_newlines, startupinfo, creationflags)
  File "/usr/lib/python2.7/subprocess.py", line 710, in __init__
    errread, errwrite)
  File "/usr/lib/python2.7/subprocess.py", line 1231, in _execute_child
    self.pid = os.fork()
OSError: [Errno 11] Resource temporarily unavailable


  [5/14] /_mozilla/mozilla/mozbrowser/mozbrowser_iframe_parentage.html
      1 [main] python2.7 3928 fork: child -1 - forked process 5232 died unexpectSTDERR: ['/e/files/dev/mozilla/servo/target/release/servo', '--cpu', '--hard-fail', '-u', 'Servo/wptrunner', '-Z', 'replace-surrogates', '-z', u'http://web-platform.test:8000/_mozilla/mozilla/mozbrowser/mozbrowser_iframe_parentage.html', '--user-stylesheet', '/e/files/dev/mozilla/servo/resources/ahem.css', '--pref', u'dom.mozbrowser.enabled=true']
  ▶ ERROR [expected OK] /_mozilla/mozilla/mozbrowser/mozbrowser_iframe_parentage.html
  └   → Traceback (most recent call last):
  File "/e/files/dev/mozilla/servo/tests/wpt/harness/wptrunner/executors/base.py", line 149, in run_test
    result = self.do_test(test)
  File "/e/files/dev/mozilla/servo/tests/wpt/harness/wptrunner/executors/executorservo.py", line 100, in do_test
    self.proc.run()
  File "/e/files/dev/mozilla/servo/python/_virtualenv/lib/python2.7/site-packages/mozprocess/processhandler.py", line 700, in run
    self.proc = self.Process([self.cmd] + self.args, **args)
  File "/e/files/dev/mozilla/servo/python/_virtualenv/lib/python2.7/site-packages/mozprocess/processhandler.py", line 103, in __init__
    universal_newlines, startupinfo, creationflags)
  File "/usr/lib/python2.7/subprocess.py", line 710, in __init__
    errread, errwrite)
  File "/usr/lib/python2.7/subprocess.py", line 1231, in _execute_child
    self.pid = os.fork()
OSError: [Errno 11] Resource temporarily unavailable


  [6/14] /_mozilla/mozilla/mozbrowser/mozbrowser_loadevents.html
         /_mozilla/mozilla/mozbrowser/mozbrowsericonchange_event.html
      0 [main] python2.7 1172 fork: child -1 - forked process 5956 died unexpectedly, retry 0, exit code 0xC0000005, errno 11
STDERR: ['/e/files/dev/mozilla/servo/target/release/servo', '--cpu', '--hard-fail', '-u', 'Servo/wptrunner', '-Z', 'replace-surrogates', '-z', u'http://web-platform.test:8000/_mozilla/mozilla/mozbrowser/mozbrowser_loadevents.html', '--user-stylesheet', '/e/files/dev/mozilla/servo/resources/ahem.css', '--pref', u'dom.mozbrowser.enabled=true']
STDERR: ['/e/files/dev/mozilla/servo/target/release/servo', '--cpu', '--hard-fail', '-u', 'Servo/wptrunner', '-Z', 'replace-surrogates', '-z', u'http://web-platform.test:8000/_mozilla/mozilla/mozbrowser/mozbrowsericonchange_event.html', '--user-stylesheet', '/e/files/dev/mozilla/servo/resources/ahem.css', '--pref', u'dom.mozbrowser.enabled=true']
  ▶ ERROR [expected OK] /_mozilla/mozilla/mozbrowser/mozbrowsericonchange_event.html
  └   → Traceback (most recent call last):
  File "/e/files/dev/mozilla/servo/tests/wpt/harness/wptrunner/executors/base.py", line 149, in run_test
    result = self.do_test(test)
  File "/e/files/dev/mozilla/servo/tests/wpt/harness/wptrunner/executors/executorservo.py", line 100, in do_test
    self.proc.run()
  File "/e/files/dev/mozilla/servo/python/_virtualenv/lib/python2.7/site-packages/mozprocess/processhandler.py", line 700, in run
    self.proc = self.Process([self.cmd] + self.args, **args)
  File "/e/files/dev/mozilla/servo/python/_virtualenv/lib/python2.7/site-packages/mozprocess/processhandler.py", line 103, in __init__
    universal_newlines, startupinfo, creationflags)
  File "/usr/lib/python2.7/subprocess.py", line 710, in __init__
    errread, errwrite)
  File "/usr/lib/python2.7/subprocess.py", line 1231, in _execute_child
    self.pid = os.fork()
OSError: [Errno 11] Resource temporarily unavailable


  [7/14] /_mozilla/mozilla/mozbrowser/mozbrowser_loadevents.html
         /_mozilla/mozilla/mozbrowser/mozbrowserlocationchange_event.html
  ▶ ERROR [expected OK] /_mozilla/mozilla/mozbrowser/mozbrowser_loadevents.html
  └   → Traceback (most recent call last):
  File "/e/files/dev/mozilla/servo/tests/wpt/harness/wptrunner/executors/base.py", line 149, in run_test
    result = self.do_test(test)
  File "/e/files/dev/mozilla/servo/tests/wpt/harness/wptrunner/executors/executorservo.py", line 100, in do_test
    self.proc.run()
  File "/e/files/dev/mozilla/servo/python/_virtualenv/lib/python2.7/site-packages/mozprocess/processhandler.py", line 700, in run
    self.proc = self.Process([self.cmd] + self.args, **args)
  File "/e/files/dev/mozilla/servo/python/_virtualenv/lib/python2.7/site-packages/mozprocess/processhandler.py", line 103, in __init__
    universal_newlines, startupinfo, creationflags)
  File "/usr/lib/python2.7/subprocess.py", line 710, in __init__
    errread, errwrite)
  File "/usr/lib/python2.7/subprocess.py", line 1231, in _execute_child
    self.pid = os.fork()
OSError: [Errno 11] Resource temporarily unavailable


STDERR: ['/e/files/dev/mozilla/servo/target/release/servo', '--cpu', '--hard-fail', '-u', 'Servo/wptrunner', '-Z', 'replace-surrogates', '-z', u'http://web-platform.test:8000/_mozilla/mozilla/mozbrowser/mozbrowserlocationchange_event.html', '--user-stylesheet', '/e/files/dev/mozilla/servo/resources/ahem.css', '--pref', u'dom.mozbrowser.enabled=true']
  [8/14] /_mozilla/mozilla/mozbrowser/mozbrowserlocationchange_event.html
         /_mozilla/mozilla/mozbrowser/mozbrowsersecuritychange_event.html
  ▶ ERROR [expected OK] /_mozilla/mozilla/mozbrowser/mozbrowserlocationchange_event.html
  └   → Traceback (most recent call last):
  File "/e/files/dev/mozilla/servo/tests/wpt/harness/wptrunner/executors/base.py", line 149, in run_test
    result = self.do_test(test)
  File "/e/files/dev/mozilla/servo/tests/wpt/harness/wptrunner/executors/executorservo.py", line 100, in do_test
    self.proc.run()
  File "/e/files/dev/mozilla/servo/python/_virtualenv/lib/python2.7/site-packages/mozprocess/processhandler.py", line 700, in run
    self.proc = self.Process([self.cmd] + self.args, **args)
  File "/e/files/dev/mozilla/servo/python/_virtualenv/lib/python2.7/site-packages/mozprocess/processhandler.py", line 103, in __init__
    universal_newlines, startupinfo, creationflags)
  File "/usr/lib/python2.7/subprocess.py", line 710, in __init__
    errread, errwrite)
  File "/usr/lib/python2.7/subprocess.py", line 1231, in _execute_child
    self.pid = os.fork()
OSError: [Errno 11] Resource temporarily unavailable


STDERR: ['/e/files/dev/mozilla/servo/target/release/servo', '--cpu', '--hard-fail', '-u', 'Servo/wptrunner', '-Z', 'replace-surrogates', '-z', u'http://web-platform.test:8000/_mozilla/mozilla/mozbrowser/mozbrowsersecuritychange_event.html', '--user-stylesheet', '/e/files/dev/mozilla/servo/resources/ahem.css', '--pref', u'dom.mozbrowser.enabled=true']
  [9/14] /_mozilla/mozilla/mozbrowser/mozbrowsershowmodalprompt_event.html
         /_mozilla/mozilla/mozbrowser/mozbrowsersecuritychange_event.html
  ▶ ERROR [expected OK] /_mozilla/mozilla/mozbrowser/mozbrowsersecuritychange_event.html
  └   → Traceback (most recent call last):
  File "/e/files/dev/mozilla/servo/tests/wpt/harness/wptrunner/executors/base.py", line 149, in run_test
    result = self.do_test(test)
  File "/e/files/dev/mozilla/servo/tests/wpt/harness/wptrunner/executors/executorservo.py", line 100, in do_test
    self.proc.run()
  File "/e/files/dev/mozilla/servo/python/_virtualenv/lib/python2.7/site-packages/mozprocess/processhandler.py", line 700, in run
    self.proc = self.Process([self.cmd] + self.args, **args)
  File "/e/files/dev/mozilla/servo/python/_virtualenv/lib/python2.7/site-packages/mozprocess/processhandler.py", line 103, in __init__
    universal_newlines, startupinfo, creationflags)
  File "/usr/lib/python2.7/subprocess.py", line 710, in __init__
    errread, errwrite)
  File "/usr/lib/python2.7/subprocess.py", line 1231, in _execute_child
    self.pid = os.fork()
OSError: [Errno 11] Resource temporarily unavailable


STDERR: ['/e/files/dev/mozilla/servo/target/release/servo', '--cpu', '--hard-fail', '-u', 'Servo/wptrunner', '-Z', 'replace-surrogates', '-z', u'http://web-platform.test:8000/_mozilla/mozilla/mozbrowser/mozbrowsershowmodalprompt_event.html', '--user-stylesheet', '/e/files/dev/mozilla/servo/resources/ahem.css', '--pref', u'dom.mozbrowser.enabled=true']
  ▶ ERROR [expected OK] /_mozilla/mozilla/mozbrowser/mozbrowsershowmodalprompt_event.html
  └   → Traceback (most recent call last):
  File "/e/files/dev/mozilla/servo/tests/wpt/harness/wptrunner/executors/base.py", line 149, in run_test
    result = self.do_test(test)
  File "/e/files/dev/mozilla/servo/tests/wpt/harness/wptrunner/executors/executorservo.py", line 100, in do_test
    self.proc.run()
  File "/e/files/dev/mozilla/servo/python/_virtualenv/lib/python2.7/site-packages/mozprocess/processhandler.py", line 700, in run
    self.proc = self.Process([self.cmd] + self.args, **args)
  File "/e/files/dev/mozilla/servo/python/_virtualenv/lib/python2.7/site-packages/mozprocess/processhandler.py", line 103, in __init__
    universal_newlines, startupinfo, creationflags)
  File "/usr/lib/python2.7/subprocess.py", line 710, in __init__
    errread, errwrite)
  File "/usr/lib/python2.7/subprocess.py", line 1231, in _execute_child
    self.pid = os.fork()
OSError: [Errno 11] Resource temporarily unavailable


  [11/14] /_mozilla/mozilla/mozbrowser/redirect.html
          /_mozilla/mozilla/mozbrowser/mozbrowsertitlechangedeagerly_event.html
STDERR: ['/e/files/dev/mozilla/servo/target/release/servo', '--cpu', '--hard-fail', '-u', 'Servo/wptrunner', '-Z', 'replace-surrogates', '-z', u'http://web-platform.test:8000/_mozilla/mozilla/mozbrowser/mozbrowsertitlechangedeagerly_event.html', '--user-stylesheet', '/e/files/dev/mozilla/servo/resources/ahem.css', '--pref', u'dom.mozbrowser.enabled=true']
  [11/14] /_mozilla/mozilla/mozbrowser/redirect.html
          /_mozilla/mozilla/mozbrowser/mozbrowsertitlechangedeagerly_event.html
  ▶ ERROR [expected OK] /_mozilla/mozilla/mozbrowser/mozbrowsertitlechangedeagerly_event.html
  └   → Traceback (most recent call last):
  File "/e/files/dev/mozilla/servo/tests/wpt/harness/wptrunner/executors/base.py", line 149, in run_test
    result = self.do_test(test)
  File "/e/files/dev/mozilla/servo/tests/wpt/harness/wptrunner/executors/executorservo.py", line 100, in do_test
    self.proc.run()
  File "/e/files/dev/mozilla/servo/python/_virtualenv/lib/python2.7/site-packages/mozprocess/processhandler.py", line 700, in run
    self.proc = self.Process([self.cmd] + self.args, **args)
  File "/e/files/dev/mozilla/servo/python/_virtualenv/lib/python2.7/site-packages/mozprocess/processhandler.py", line 103, in __init__
    universal_newlines, startupinfo, creationflags)
  File "/usr/lib/python2.7/subprocess.py", line 710, in __init__
    errread, errwrite)
  File "/usr/lib/python2.7/subprocess.py", line 1231, in _execute_child
    self.pid = os.fork()
OSError: [Errno 11] Resource temporarily unavailable


STDERR: ['/e/files/dev/mozilla/servo/target/release/servo', '--cpu', '--hard-fail', '-u', 'Servo/wptrunner', '-Z', 'replace-surrogates', '-z', u'http://web-platform.test:8000/_mozilla/mozilla/mozbrowser/redirect.html', '--user-stylesheet', '/e/files/dev/mozilla/servo/resources/ahem.css', '--pref', u'dom.mozbrowser.enabled=true']
  ▶ ERROR [expected OK] /_mozilla/mozilla/mozbrowser/redirect.html
  └   → Traceback (most recent call last):
  File "/e/files/dev/mozilla/servo/tests/wpt/harness/wptrunner/executors/base.py", line 149, in run_test
    result = self.do_test(test)
  File "/e/files/dev/mozilla/servo/tests/wpt/harness/wptrunner/executors/executorservo.py", line 100, in do_test
    self.proc.run()
  File "/e/files/dev/mozilla/servo/python/_virtualenv/lib/python2.7/site-packages/mozprocess/processhandler.py", line 700, in run
    self.proc = self.Process([self.cmd] + self.args, **args)
  File "/e/files/dev/mozilla/servo/python/_virtualenv/lib/python2.7/site-packages/mozprocess/processhandler.py", line 103, in __init__
    universal_newlines, startupinfo, creationflags)
  File "/usr/lib/python2.7/subprocess.py", line 710, in __init__
    errread, errwrite)
  File "/usr/lib/python2.7/subprocess.py", line 1231, in _execute_child
    self.pid = os.fork()
OSError: [Errno 11] Resource temporarily unavailable


  [13/14] /_mozilla/mozilla/mozbrowser/reload.html
      0 [main] python2.7 7792 fork: child -1 - forked process 2680 died unexpectSTDERR: ['/e/files/dev/mozilla/servo/target/release/servo', '--cpu', '--hard-fail', '-u', 'Servo/wptrunner', '-Z', 'replace-surrogates', '-z', u'http://web-platform.test:8000/_mozilla/mozilla/mozbrowser/reload.html', '--user-stylesheet', '/e/files/dev/mozilla/servo/resources/ahem.css', '--pref', u'dom.mozbrowser.enabled=true']
  ▶ ERROR [expected OK] /_mozilla/mozilla/mozbrowser/reload.html
  └   → Traceback (most recent call last):
  File "/e/files/dev/mozilla/servo/tests/wpt/harness/wptrunner/executors/base.py", line 149, in run_test
    result = self.do_test(test)
  File "/e/files/dev/mozilla/servo/tests/wpt/harness/wptrunner/executors/executorservo.py", line 100, in do_test
    self.proc.run()
  File "/e/files/dev/mozilla/servo/python/_virtualenv/lib/python2.7/site-packages/mozprocess/processhandler.py", line 700, in run
    self.proc = self.Process([self.cmd] + self.args, **args)
  File "/e/files/dev/mozilla/servo/python/_virtualenv/lib/python2.7/site-packages/mozprocess/processhandler.py", line 103, in __init__
    universal_newlines, startupinfo, creationflags)
  File "/usr/lib/python2.7/subprocess.py", line 710, in __init__
    errread, errwrite)
  File "/usr/lib/python2.7/subprocess.py", line 1231, in _execute_child
    self.pid = os.fork()
OSError: [Errno 11] Resource temporarily unavailable


Ran 14 tests finished in 4.0 seconds.
  • 0 ran as expected. 0 tests skipped.
  • 14 tests had errors unexpectedly
```

**Comment by Coder206** [comment]
@nox Squash completed! :) Learning lots about Git!!

**Comment by Coder206** [comment]
@Wafflespeanut Thanks :)

**Comment by asajeffrey** [comment]
Yay, it merged, thanks for all the work @Coder206!

**Comment by Coder206** [comment]
@asajeffrey Thanks for your help as well :)

**Pull request merged**